spec/javascripts/govuk-component/feedback-spec.js

```diff
@@ -0,0 +1,705 @@
+
+            '<input type="hidden" name="url" value="http://example.com/path/to/page">' +
+            '<input type="hidden" name="user_agent" value="Safari">' +
+
+            '<h2 class="pub-c-feedback__form-heading">Help us improve GOV.UK</h2>' +
+            '<p class="pub-c-feedback__form-paragraph">To help us improve GOV.UK, we\'d like to know more about your visit today. We\'ll send you a link to a feedback form. It will take only 2 minutes to fill in. Don\'t worry we won\'t send you spam or share your email address with anyone.</p>' +
+
+            '<div class="pub-c-feedback__form-group js-form-group">' +
+              '<label class="pub-c-feedback__form-label" for="email-field">Email address</label>' +
+              '<input id="email-field" class="pub-c-feedback__form-field" type="text" name="email_survey_signup[email_address]">' +
+            '</div>' +
+
+            '<input class="pub-c-feedback__submit" type="submit" value="Send me the survey">' +
+            '<a href="FIXME" class="">Don\'t have an email address?</a>' +
+          '</div>' +
+        '</div>' +
+      '</form>' +
+    '</div>';
+
+  beforeEach(function () {
+    setFixtures(FIXTURE);
+  });
+
+  it("hides the forms", function () {
+    loadFeedbackComponent();
+
+    expect($('.pub-c-feedback .js-feedback-form')).toHaveClass('js-hidden');
+  });
+
+  it("shows the prompt", function () {
+    loadFeedbackComponent();
+
+    expect($('.pub-c-feedback .js-prompt')).not.toHaveClass('js-hidden');
+    expect($('.pub-c-feedback .js-prompt-questions')).not.toHaveClass('js-hidden');
+  });
+
+  it("conveys that the feedback forms are hidden", function() {
+    loadFeedbackComponent();
+
+    expect($('.js-feedback-form#something-is-wrong').attr('aria-hidden')).toBe('true');
+    expect($('.js-feedback-form#page-is-not-useful').attr('aria-hidden')).toBe('true');
+  });
+
+  it("conveys that the form is not expanded", function () {
+    loadFeedbackComponent();
+
+    expect($('.js-toggle-form[aria-controls="page-is-not-useful"]').attr('aria-expanded')).toBe('false');
+    expect($('.js-toggle-form[aria-controls="something-is-wrong"]').attr('aria-expanded')).toBe('false');
+  });
+
+  describe("clicking the 'page was useful' link", function () {
+    it("displays a success message", function () {
+      loadFeedbackComponent();
+      $('a.js-page-is-useful').click();
+
+      var $success = $('.js-prompt-success');
+
+      expect($success).not.toHaveClass('js-hidden');
+      expect($success).toHaveText("Thanks for your feedback.");
+    });
+
+    it("hides the question links", function () {
+      loadFeedbackComponent();
+      $('a.js-page-is-useful').click();
+
+      expect($('.js-prompt-questions')).toHaveClass('js-hidden');
+    });
+
+    it("triggers a Google Analytics event", function () {
+      var analytics = {
+        trackEvent: function() {}
+      };
+
+      withGovukAnalytics(analytics, function () {
```

Comment:
Not sure why the `withGovukAnalytics` method is necessary. eg Track click spec seems to work ok: https://github.com/alphagov/static/blob/master/spec/javascripts/modules/track-click.spec.js

Your commit doesn't explain why.

Reply:
I think this is probably a hangover from the SM code. You're right though, removing it is fine, tests still work, thanks.